features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1003,4 +1040,9 @@
         </ExemptContext>
     </IntermediateCertValidation>
 
+    <ConfigurationStore>
```

Comment:
Put a description regarding this configuration. What it is and why it is using...!!!